ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TopToolbarView.swift

```diff
@@ -751,10 +753,10 @@ class TopToolbarView: UIView, ToolbarProtocol {
     if cancelButton.isHidden == inOverlayMode {
       cancelButton.isHidden = !inOverlayMode
     }
-    backButton.isHidden = !toolbarIsShowing || inOverlayMode
-    forwardButton.isHidden = !toolbarIsShowing || inOverlayMode
-    shareButton.isHidden = !toolbarIsShowing || inOverlayMode
-    trailingItemsStackView.isHidden = !toolbarIsShowing || inOverlayMode
+    backButton.isHidden = toolbarIsShowing || inOverlayMode
+    forwardButton.isHidden = toolbarIsShowing || inOverlayMode
+    shareButton.isHidden = toolbarIsShowing || inOverlayMode
+    trailingItemsStackView.isHidden = toolbarIsShowing || inOverlayMode
```

Comment:
we are hiding all those buttons when `toolbarIsShowing` == true now?

Reply:
Yeah that's supposed to be the correct logic, for some reason the bool was flipped when setting, this doesn't actually change anything just corrects the setter